IPython/utils/_process_cli.py

```diff
@@ -11,16 +11,11 @@
 in the details. 
 """
 
-# Import cli libraries:
-import clr
-import System
 
-# Import Python libraries:
+import System
 import os
-
-# Import IPython libraries:
 from IPython.utils import py3compat
-from ._process_common import arg_split
```

Comment:
This needs to be there - `process.py` expects to be able to import it from this file.